key/soft_key.go

```diff
@@ -51,6 +49,8 @@ const (
 	EwoqPrivateKey = privKeyEncPfx + rawEwoqPk
 )
 
+var ewoqKeyBytes = []byte("56289e99c94b6912bfc12adc093c9b51124f0dc54ac7a766b2bc5ccf558d8027")
```

Comment:
My bet is most people outside of platform have no idea what the ewoq key means. If we're going to create a constant with respect to this SDK for this, let's be more descriptive and document it. The SDK should be more user friendly than avalanchego. I only know about this because I used to use Gecko!

Reply:
deleted this since it was only used in test files